Makefile

```diff
@@ -0,0 +1,20 @@
+APP:= brasil-api
+TAG:= 1.0.0
+
+# DEV
+docker-build-local:
+	docker build -q -t ${APP}:${TAG}-local .
+
+docker-run-local:
+	docker run --entrypoint=ash -p 3000:3000 ${APP}:${TAG}-local -c 'npm run dev'
+
+local: docker-build-local docker-run-local
+
+# PRD
+docker-build:
+	docker build -q -t ${APP}:${TAG} .
+
+docker-run:
+	docker run -p 3000:3000 ${APP}:${TAG}
+
+docker: docker-build docker-run
```

Comment:
Sou meio bixado com esses arquivos `Makefile`. Como isso é coisa de sistema Unix, não sei como fica o comportamento no Windows.

Pra mim, isso tanto faz.

Reply:
Sim, isso é mais para quem corre Mac ou Linux, e não atrapalha quem não faz uso.

Reply:
@emersonmx [quanto ao make no windows](https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows)